NewForce-at-Mountwest/bangazon-workforce-farfalle
# Assign an employee to a training program

# Description
Users now have the ability to assign an employee to a training program from the employee details page


## Related Ticket(s)
https://trello.com/c/zVYHAknV/17-hr-should-be-able-to-assign-employees-to-training-programs


## Steps to Test
* Add and commit on your own branch before checking out to sw-assignEmployeeToTraining
* Pull down the most recent version and start your server
*  Navigate to the 'training programs' tab and make sure there are upcoming training programs.  If not, add one. 
* Navigate to 'employees' and click on the details link of any employee
* click 'assign to training program'
*You will be presented with a new screen that has the employee's name and a dropdown form that includes future training programs that still have availability, and the employee is not already registered.
* select a training program and click 'assign.'  You will be redirected back to the employee's detail view and should see the training program added to their list.
*If you follow the previous steps, you should NOT see the program you just added in the dropdown.

# Checklist:
